src/components/icons/export.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default ({ className }) => (
+  <svg
+    className={className}
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24">
+    <path d="M5 16v4h14v-4h2v4c0 1-.8 2-1.9 2H5a2 2 0 01-2-2v-4h2zm8-14v11l3.5-3.5L18 11l-6 6-6-6 1.5-1.5L11 13V2h2z" />
+  </svg>
+);
```

Comment:
In create react app you can load SVGs without wrapping them in a component e.g.

```
import { ReactComponent as ExportIcon } from './export.svg';

const App = () => (
  <div>
    {/* Icon is an actual React component */}
    <ExportIcon />
  </div>
);

```

Reply:
Yep that's what I had originally, but I had to remove it because it broke in Studio, which doesn't support webpack's inline SVG loader. See #75.